src/overlays/actors/ovl_En_Zob/z_en_zob.c

```diff
@@ -588,7 +588,7 @@ void func_80BA0728(EnZob* this, PlayState* play) {
     sp28.x = this->actor.projectedPos.x;
     sp28.y = this->actor.projectedPos.y;
     sp28.z = this->actor.projectedPos.z;
-    func_801A1FB4(3, &sp28, 0x6C, 1000.0f);
+    func_801A1FB4(3, &sp28, NA_BGM_BASS_PLAY, 1000.0f);
```

Comment:
```suggestion
    func_801A1FB4(SEQ_PLAYER_BGM_SUB, &sp28, NA_BGM_BASS_PLAY, 1000.0f);
```
While you're filling in enums for `func_801A1FB4`, the first arg also has an enum that needs filling. Could you quickly do that for the rest of the `func_801A1FB4`? I could always do it later though when this function is named

Reply:
Yep! I can take care of that too.